src/Humanizer.Tests.Shared/DateToOrdinalWordsTests.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using Xunit;
+
+namespace Humanizer.Tests
+{
+    public class DateToOrdinalWordsTests
+    {
+        [Fact]
+        public void CanToOrdinalWords()
+        {
+            var date = new DateTime(2015, 1, 1);
+
+            Assert.Equal("1st January 2015", date.ToOrdinalWords());
+        }
+
+        [UseCulture("pt-BR")]
+        [Fact]
+        public void CanToOrdinalWordsIncludingGender()
+        {
+            var date = new DateTime(2015, 1, 1);
+
+            Assert.Equal("1º janeiro 2015", date.ToOrdinalWords(GrammaticalGender.Masculine));
```

Comment:
@hazzik what do you target with this question, the gender parameter?
I also thought it should be tied to a default value, the `Ordinalize` extensions also do it that way. By this the localization can decide which gender to pick as default.